sync/src/chain.rs

```diff
@@ -2119,12 +2160,28 @@ impl ChainSync {
 	}
 }
 
+/// Checks if peer is able to process zgp-transactions
+fn accepts_zgp_transaction(client_id: &str) -> bool {
+	// Parity versions starting from this will accept zgp-transactions
+	const ZGP_VERSION: (u32, u32) = (1u32, 6u32);
+
+	if !client_id.starts_with("Parity/v") {
+		return false;
+	}
+	let ver: Vec<u32> = client_id[8..].split('.')
```

Comment:
Don't know if checking Parity' version is necessary - seems like current Parity version doesn't drop connection, event when zgp-transaction is received => maybe this is an optional check